{{ cookiecutter.repo_name }}/README.rst

```diff
@@ -1,22 +1,104 @@
+|CI| |Codecov| |Black| |Tracking|
 
-.. image:: https://img.shields.io/pypi/v/{{ cookiecutter.repo_name }}.svg
-        :target: https://pypi.python.org/pypi/{{ cookiecutter.repo_name }}
+.. |Black| image:: https://img.shields.io/badge/code_style-black-black
+        :target: https://github.com/psf/black
+   
+.. |CI| image:: https://github.com/{{ cookiecutter.github_org }}/{{ cookiecutter.project_name }}/actions/workflows/main.yml/badge.svg
+        :target: https://github.com/{{ cookiecutter.github_org }}/{{ cookiecutter.project_name }}/actions/workflows/main.yml
 
+.. |Codecov| image:: https://codecov.io/gh/{{ cookiecutter.github_org }}/{{ cookiecutter.project_name }}/branch/main/graph/badge.svg
+        :target: https://codecov.io/gh/{{ cookiecutter.github_org }}/{{ cookiecutter.project_name }}
+   
+.. |Forge| image:: https://img.shields.io/conda/vn/conda-forge/{{ cookiecutter.project_name }}
+        :target: https://anaconda.org/conda-forge/{{ cookiecutter.project_name }}
 
-{{ cookiecutter.project_short_description}}
+.. |PR| image:: https://img.shields.io/badge/PR-Welcome-29ab47ff
+        :target: https://github.com/{{ cookiecutter.github_org }}/{{ cookiecutter.project_name }}/blob/main/CONTRIBUTING.rst
 
-* Free software: 3-clause BSD license
-* Documentation: (COMING SOON!) https://{{ cookiecutter.github_org }}.github.io/{{ cookiecutter.repo_name }}.
+.. |PyPi| image:: https://img.shields.io/pypi/v/{{ cookiecutter.project_name }}
+        :target: https://pypi.org/project/{{ cookiecutter.project_name }}/
+   
+.. |PythonVersion| image:: https://img.shields.io/pypi/pyversions/{{ cookiecutter.project_name }}
+        :target: https://pypi.org/project/{{ cookiecutter.project_name }}/
 
-Features
---------
+.. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
+        :target: https://github.com/{{ cookiecutter.github_org }}/{{ cookiecutter.project_name }}/issues
 
-* TODO
+{{ cookiecutter.project_short_description }}
+
+* LONGER DESCRIPTION HERE
+
+For more information about the {{ cookiecutter.project_name }} library, see the `users manual <https://{{ cookiecutter.github_org }}.github.io/{{ cookiecutter.project_name }}>`_.
+
+Installation
+------------
+
+The preferred method is to use `Miniconda Python
+<https://docs.conda.io/projects/miniconda/en/latest/miniconda-install.html>`_
+and install from the "conda-forge" channel of Conda packages.
+
+To add "conda-forge" to the conda channels, run the following in a terminal. ::
+
+        conda config --add channels conda-forge
+
+We want to install our packages in a suitable conda environment.
+The following creates and activates a new environment named ``{{ cookiecutter.project_name }}`` ::
+
+        conda create -n {{ cookiecutter.project_name }} python=3
+        conda activate {{ cookiecutter.project_name }}
+
+Then, to fully install ``{{ cookiecutter.project_name }}`` in our active environment, run ::
+
+        conda install {{ cookiecutter.project_name }}
```

Comment:
Change all references from `project_name` to `package_dist_name` in the installation section